bin/cmd.js

```diff
@@ -199,6 +201,14 @@ function init (_argv) {
     playerArgs.smplayer.push('-ontop')
   }
 
+  if (argv.downloadLimit > 0) {
+    argv.downloadLimit = argv.d = argv['download-limit'] = argv.downloadLimit * 1000
+  }
+
+  if (argv.uploadLimit > 0) {
+    argv.uploadLimit = argv.u = argv['upload-limit'] = argv.uploadLimit * 1000
```

Comment:
```suggestion
    argv.downloadLimit = argv.d = argv['download-limit'] = argv.downloadLimit * 1024
  }

  if (argv.uploadLimit > 0) {
    argv.uploadLimit = argv.u = argv['upload-limit'] = argv.uploadLimit * 1024
```

Reply:
if you look at https://github.com/alxhotel/speed-limiter, doc says u got to do 200 * 1000 to get 200 kB/s though.